java/algorithms/Trie-MemUnsat01/Main.java

```diff
@@ -0,0 +1,285 @@
+import org.sosy_lab.sv_benchmarks.Verifier;
+import java.util.Queue;
+
+/**
+ * Type             : Memory Safety
+ * Expected Verdict : False
+ * Last modified by : Zafer Esen <zafer.esen@it.uu.se>
+ * Date             : 9 October 2019
+ *
+ * Original license follows.
+ */
+
+/******************************************************************************
```

Comment:
What is the license here? I don't see any.

Reply:
Added license.